language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/symbol/BallerinaSymbolClientCapabilities.java

```diff
@@ -54,6 +58,22 @@ public void setType(boolean type) {
         this.type = type;
     }
 
+    public boolean isGetTypeFromExpression() {
+        return getTypeFromExpression;
```

Comment:
This method is unused. We should check if the client has this capability before processing the request right? Same for the method below this one.

Reply:
As we discussed offline, the client capability check is not needed (Since we are checking the server capabilities from the client side and initiate the request). However keeping these getters and setters for future requirements.